alex/applications/PublicTransportInfoEN/directions.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
```

Comment:
yeah, it would, however it is used a little bit differently. Czech directions.py has the "zastavka" prefix at each origin/destination for example.